src/wizard/wizard.scss

```diff
@@ -2,168 +2,147 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
-@mixin iui-wizards {
+@mixin iui-wizard {
   @include iui-reset;
   @include iui-font-family;
 
-  > .iui-wizards-wrapper {
+  > .iui-wizard-container {
     display: flex;
     flex-direction: row;
     justify-content: space-around;
     align-items: flex-start;
     width: 100%;
     margin-bottom: $iui-baseline * 0.5;
 
-    > .iui-wizards-step {
+    .iui-wizard-step {
+      flex: 1 1 0;
       align-items: center;
       display: flex;
-      flex-direction: column-reverse;
+      flex-direction: column;
       font-size: $iui-font-size-small;
```

Comment:
Maybe we have a mixin for font stuff?

Reply:
Replaced `font-size`, `font-weight`, and `line-height` with `@include iui-text-small;` in the next push.